src/components/alert/alert.svelte

```diff
@@ -68,22 +77,39 @@
       --leo-alert-background-color,
       var(--default-background)
     );
-    display: flex;
     color: var(--default-text-color, var(--leo-color-text-primary));
     padding: var(--leo-spacing-xl);
     border-radius: var(--leo-radius-m);
     gap: var(--leo-spacing-xl);
     font: var(--leo-font-default-regular);
 
+    display: grid;
+    grid-template-columns: min-content 1fr;
+
+    &:has(.content-after) {
```

Comment:
How comfortable are we with `:has()`? Looks like most concerning would be Firefox for Android not supporting it yet.

https://caniuse.com/?search=%3Ahas

Reply:
I think we're fairly comfortable with it, but @petemill to confirm.